vars/RedHat.yml

```diff
@@ -1,10 +1,9 @@
-os_version: "{{ ansible_lsb.release if ansible_lsb is defined else ansible_distribution_version }}"
-os_version_major: "{{ os_version | regex_replace('^([0-9]+)[^0-9]*.*', '\\\\1') }}"
+os_version_major: "{{ ansible_distribution_major_version }}"
 
 # EPEL repository released packaged per OS version
 epel_releases:
   '6': 'epel-release-6-8.noarch.rpm'
-  '7': 'e/epel-release-7-5.noarch.rpm'
+  '7': 'e/epel-release-7-6.noarch.rpm'
```

Comment:
https://dl.fedoraproject.org/pub/epel/7/x86_64/e/

- /epel-release-7-5.noarch.rpm is no longer available however there is new package 


> epel-release-7-6.noarch.rpm added 2016-04-01